### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by ecaquot**

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sweng/vanjel/LoginActivityTest.java [hunk 1]
```diff
@@ -41,80 +43,57 @@
...
             new ActivityTestRule<>(LoginActivity.class);
 
-    @Before
-    public void init() {
-        FirebaseAuthCustomBackend.setNullUser(true);
-        ActivityRule.finishActivity();
-        Intent i = new Intent();
-        ActivityRule.launchActivity(i);
+    @BeforeClass
+    public static void setUp() {
```
Comment (AAA97AAA) [inline app/src/androidTest/java/ch/epfl/sweng/vanjel/LoginActivityTest.java]: why did you remove @Before that were there in order to set the user as null with the mock? 
Depending on the previous test suite, it can set it with a patient or a doctor

Comment (AAA97AAA) [reply app/src/androidTest/java/ch/epfl/sweng/vanjel/LoginActivityTest.java]: well sorry, didn't see the setupNoExtras (even If I think having too much booleans in the constructor can be not as readable as multiple lines that explicitly shows what variables are set)

File: app/src/main/java/ch/epfl/sweng/vanjel/mainMenu/MainMenu.java [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+import ch.epfl.sweng.vanjel.nearbyDoctor.NearbyDoctor;
+import ch.epfl.sweng.vanjel.patientAppointment.PatientPersonalAppointments;
+import ch.epfl.sweng.vanjel.patientInfo.PatientInfo;
+import ch.epfl.sweng.vanjel.profile.Profile;
+import ch.epfl.sweng.vanjel.searchDoctor.SearchDoctor;
+import ch.epfl.sweng.vanjel.treatedPatient.TreatedPatients;
+
+/**
+ * @author Etienne CAQUOT
+ */
```
Comment (AAA97AAA) [inline app/src/main/java/ch/epfl/sweng/vanjel/mainMenu/MainMenu.java]: add me in reviewer -plz

Comment (ecaquot) [reply app/src/main/java/ch/epfl/sweng/vanjel/mainMenu/MainMenu.java]: done

File: app/src/androidTest/java/ch/epfl/sweng/vanjel/LoginActivityTest.java (hunk 1 above)
Comment (ecaquot) [reply app/src/androidTest/java/ch/epfl/sweng/vanjel/LoginActivityTest.java]: It's a helper functions so I guess it is not very important, see with it's author @ljoss17
